source/unified-test-format/unified-test-format.rst

```diff
@@ -1798,6 +1857,155 @@ Use a ``listIndexes`` command to check whether the index exists. Note that it is
 currently not possible to run ``listIndexes`` from within a transaction.
 
 
+loop
+~~~~
+
+The ``loop`` operation executes sub-operations in a loop.
+It supports the following arguments:
+
+- ``operations``: the sub-operations to run on each loop iteration.
```

Comment:
@p-mongo: Did we previously talk about advising tests not to nest `loop` operations? If so, I'd suggest adding:

> Tests SHOULD NOT include multiple loop operations (nested or sequential).

Note: this spec uses SHOULD for things it doesn't programmatically enforce. In this case, given how termination works, a test would either get stuck in the first or most-nested loop until a termination signal is received. And subsequent loops would be skipped. So there isn't a technical problem with having multiple loops per test. Rather, it just doesn't make sense.

I don't think we need to say "Test files SHOULD NOT" instead. If multiple tests in the file each use a single loop operation, that's fine, since tests might be skipped. Obviously, the runner wouldn't be able to execute both tests to completion, though.

Reply:
The language in DAT mandates exactly one loop but there aren't any constraints in unified test format. I added your suggested text.